ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -592,9 +592,8 @@ export default class ConfirmTransactionBase extends Component {
 
     this._removeBeforeUnload();
     updateCustomNonce('');
-    cancelTransaction(txData).then(() => {
-      history.push(mostRecentOverviewPage);
-    });
+    await cancelTransaction(txData);
+    history.push(mostRecentOverviewPage);
```

Comment:
why was this change necessary?

Reply:
IMHO, the modified code using await provides a clearer and more concise way of handling this asynchronous operation

Reply:
In general we should try to avoid changes unnecessary to the PR, as there may be unintended consequences that then get traced down to this. This is part of our "Auditability" engineering principle. I'm going to approve and just leave this is a critique to apply to future prs.